expansion/src/test/java/no/unit/nva/expansion/ResourceExpansionServiceTest.java

```diff
@@ -188,6 +214,63 @@ private static UserInstance sampleSender() {
         return new UserInstance(SOME_SENDER, SOME_ORG);
     }
 
+    private void assertThatExpandedDoiRequestContainsOnlyDoiRequestMessagesAndNotAnyOfTheSupportMessages(
+        PublicationWithDoiRequestAndAllTypesOfMessages samplePublication,
+        ExpandedDoiRequest expandedDoiRequestConversation) {
+        var messagesIdentifiersInExpandedDoiRequest =
+            extractMessageIdenfiersFromExpandedDoiRequest(expandedDoiRequestConversation);
+
+        assertThat(messagesIdentifiersInExpandedDoiRequest,
+                   contains(samplePublication.getDoiRequestMessageIdentifiers()));
+        assertThat(Arrays.asList(samplePublication.getSupportMessageIdentifiers()),
```

Comment:
Unfortunately, hamcrest does not have an easy way to assert that the actual collection does not have any of the items of the non-expected collection. So we have to invert the test, assert that none of the non-desired items are in the actual collection.